galley/cmd/galley/cmd/root.go

```diff
@@ -82,6 +82,10 @@ func GetRootCmd(args []string, printf, fatalf shared.FormatFn) *cobra.Command {
 			if !serverArgs.EnableServer && !validationArgs.EnableValidation {
 				fatalf("Galley must be running under at least one mode: server or validation")
 			}
+			if validationArgs.EnableValidation && validationArgs.WebhookConfigFile == "" {
```

Comment:
This should be an error instead. We should have a .Validate() method on validationArgs and check for such inconsistencies.

Reply:
Hmm, I meant to, but on the other hand I'm afraid that would break backwards compatibility (as we didn't consider this an error before).